src/utils/ContractUtils.ts

```diff
@@ -112,5 +112,8 @@ export const checkLinkValidity = (link: string): boolean => {
 
 export const getNetworkName = (networkId: string): string => {
```

Comment:
get the chain id here instead of from parameter

Reply:
Done